### ISSUE TITLE
Partial border for recipient select

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/mail#9445]
### Steps to reproduce

1. Set up an account
2. Click *New message*
3. Focus the *To* field

### Expected behavior

Good design

### Actual behavior

![image](https://github.com/nextcloud/mail/assets/1374172/2dd1c671-2db0-4e67-95e3-f3a33227f21b)

### Mail app version

3.6.0 beta1

### PHP engine version

None

### Web server

None

### Database

None

### Additional info

@GretaD @hamza221 how can we fix this?

**Comment by GretaD** [comment]
> @GretaD @hamza221 how can we fix this?

I cannot think of any reasonable fix or even a hack how to fix this. 
I was thinking having a full square with borders, but even as a hack its not fully doable, because:

![Screenshot from 2024-03-15 17-28-34](https://github.com/nextcloud/mail/assets/12728974/6930096c-2bd3-433c-9c5f-9ce00c1a7c76)

![Screenshot from 2024-03-15 17-29-34](https://github.com/nextcloud/mail/assets/12728974/3d67f662-4b1c-456b-8f98-4decb40bb62c)

**Comment by thstyl2000** [comment]
Sorry to bump, but it seems to me that this still looks quite unusual: clicking on the "To:" field directly opens the drop-down with "No resutls", although one has not done any search yet! Clicking somewhere else, and then back again on the "To:" field, opens the drop-down with an arbitrary list of contacts.

Suggestion: do not open the drop-down on click/focus, but only after typing sth, OR: open drop-down with full (alphabetically or frequency or importance sorted) list.

**Comment by ChristophWurst** [comment]
@thstyl2000 open a new ticket. This is very unrelated to the visual problem reported here.
